src/main/java/edu/phema/elm_to_omop/vocabulary/translate/map/ActCodeToVisitConceptMap.java

```diff
@@ -0,0 +1,45 @@
+package edu.phema.elm_to_omop.vocabulary.translate.map;
+
+import edu.phema.elm_to_omop.vocabulary.phema.PhemaCode;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Translates the ActCode vocabulary to the OMOP internal visit concept vocabulary
+ * <p>
+ * Source: https://www.hl7.org/fhir/STU3/v3/ActEncounterCode/vs.html
+ * Target: http://athena.ohdsi.org/search-terms/terms?vocabulary=Visit
+ */
+public class ActCodeToVisitConceptMap implements IVocabularyMap {
+    private static Map<String, String> map;
+
+    static {
+        map = new HashMap<>();
+
+        // TODO: Have someone validate these mappings
+        map.put("AMB", "OP");
+        map.put("EMER", "ER");
+        map.put("FLD", "LCTP");
+        map.put("HH", "OMOP4822459");
+        map.put("IMP", "IP");
+        map.put("ACUTE", "IP");
+        map.put("NONAC", "IP");
+        map.put("PRENC", "OP");
+        map.put("SS", "OP");
+        map.put("VR", "OP");
```

Comment:
Maybe telehealth? http://athena.ohdsi.org/search-terms/terms/5083

Reply:
We need to use the `Visit` vocabulary here unfortunately. I know this is limiting, but I'm going to leave this code for now.